.github/workflows/cicd.yml

```diff
@@ -38,15 +38,15 @@ jobs:
     if: github.event_name == 'schedule' || github.event_name == 'pull_request' || github.event_name == 'push'
     uses: ./.github/workflows/reusable-build.yml
     with:
-      build_artifact: Build-x64
+      build_artifact: Build-x64-vcxproj
       generate_release_package: true
 
   cmake:
     # Always run this job.
     if: github.event_name == 'schedule' || github.event_name == 'pull_request' || github.event_name == 'push'
     uses: ./.github/workflows/reusable-cmake-build.yml
     with:
-      build_artifact: Build-x64-cmake
+      build_artifact: Build-x64
```

Comment:
Why the change on this line?  Where does the string come from?

Reply:
This is to test if the tests work with the output of the cmake build and not the regular build. In a subsequent PR all the checked in vcxproj files and this CI task itself will be removed.